package.json

```diff
@@ -56,6 +57,7 @@
     "glamor": "^2.20.37",
     "jest-glamor-react": "^3.2.2",
     "kcd-scripts": "^0.31.1",
+    "node": "^8.9.4",
```

Comment:
Great idea adding this, hopefully it should make contributing a bit easier and cut down on issues around node versions 🙂.